app/api/auth/serializers.py

```diff
@@ -0,0 +1,141 @@
+import typing as tp
+
+from django.core.validators import validate_email
+from django.contrib.auth.hashers import make_password
+from django.contrib.auth import get_user_model, password_validation
+
+from rest_framework import serializers
+from rest_framework_simplejwt.serializers import TokenObtainPairSerializer
+
+from api.auth.validation import validate_username_field, validate_email_field, validate_password_field
+from base import settings
+
+
+class RegisterUserSerializer(serializers.Serializer):
+    """
+    Serializer for register new user
+    """
+
+    username = serializers.CharField(max_length=100, required=True)
+    password = serializers.CharField(max_length=500, required=True)
```

Comment:
why that much?

Reply:
yes, max length for password is really too much, i forgot this moment

Reply:
done